proposals/extend_pod_security_admission.md

```diff
@@ -72,114 +71,149 @@ For example, if a user wants to apply Restricted PSS to the selected namespaces
+    match:
+      any:
+        - resources:
+            kinds:
+              - Pod
+          namespaces:
+            - test
+            - staging
     validate:
       # this new type of rule only deals with PSS profiles
       # as we need to check if the value in the resource
       # is actually allowed by PSS
       podSecurity:
-         profile: restricted
-         # version must be a valid Kubernetes minor version,
-         # or `latest`.
-         version: latest
-         exclude:
-           # controlName is the Control defined in PSS
-           # +required
-         - controlName: Capabilities
-           images:
-           - ghcr.io/example/nginx:1.2.3
-           # +optional
-           restrictedField: spec.containers[*].securityContext.capabilities.add
-           # Any type
-           # +optional
-           values: # how to define an object?
-           - "KILL"
-           - "SETGID"
-           - "SETUID"
+        profile: restricted
+        # version must be a valid Kubernetes minor version,
+        # or `latest`.
+        version: v1.24
+        exclude:
+          # controlName is the Control defined in PSS
+          # +required
+          - controlName: Volumes
+            labels:
+              env: prod
 ```
 
-Note that securityContext is configured at container level, not pod, as the matching image selects a specific container. The pod creation will be rejected if there's any violation found in its container. For pod level checks, other match criteria such as labels could be used.
+### Policy Reports
 
 For a PolicyReport, `cpol.rules.validate.podSecurity.exclude.restrictedField` and `cpol.rules.validate.podSecurity.exclude.controlName` will be added to `polr.results.properties` to audit the result.
 
 ```yaml
 PolicyReport:
 results:
-- message: |- 
-    'validation error: The label `app.kubernetes.io/name` is required. 
-    Rule check-for-labels failed at path
-    /metadata/labels/app.kubernetes.io/name/'
-  policy: require-labels
-  resources:
-  - apiVersion: v1
-    kind: Pod
-    name: nginx
-    namespace: default
-    uid: 13fcb726-5597-4fcd-a708-ce8b558bd484
-  rule: check-for-labels
-  properties:
-    controlName: Capabilities
-    podSecurity: spec.containers[*].securityContext.capabilities.add
-  scored: true
-  status: fail
+  - message: |-
+      'validation error: The label `app.kubernetes.io/name` is required. 
+      Rule check-for-labels failed at path
+      /metadata/labels/app.kubernetes.io/name/'
+    policy: require-labels
+    resources:
+      - apiVersion: v1
+        kind: Pod
+        name: nginx
+        namespace: default
+        uid: 13fcb726-5597-4fcd-a708-ce8b558bd484
+    rule: check-for-labels
+    properties:
+      controlName: Capabilities
+      podSecurity: spec.containers[*].securityContext.capabilities.add
+    scored: true
+    status: fail
 ```
 
-### Exceptions for container security context
+### Fields outside container object
```

Comment:
Still trying to think of a better title here....

Maybe "Additional pod security controls"? And an sentence to explain:

Here are the pod security controls that are not declared at the container-level: